docs/source/cli_help/list.rst

```diff
@@ -16,8 +16,10 @@ Lists all branches that fall into one of the specified categories:
     * ``childless``: all branches that do not possess child branches,
     * ``managed``: all branches that appear in the definition file,
     * ``slidable``: all managed branches that have an upstream and can be slid out with :ref:`slide-out` command
-    * ``slidable-after <branch>``: the downstream branch of the <branch>, if it exists and is the only downstream of <branch> (i.e. the one that can be slid out immediately following <branch>),
+    * ``slidable-after <branch>``: the downstream branch of the <branch>, if it exists and is the only downstream of <branch>
+      (i.e. the one that can be slid out immediately following <branch>),
```

Comment:
Nit: would it work if the second lines in each bullet point are more indented? so that all description lines are aligned (start in the same column)

Reply:
Same for `go`

Reply:
@PawelLipski hmmm, it would have to involve some `if` programming, dependent upon the `:` character which in turn would mean that we would have to be extra careful in using `:` inside bullet point lists and I don't it's worth it, wdyt?

Reply:
Okay, up to you judgement, it's already clear (even if not perfectly formatted)